CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Change Log
 
+## [0.28.0](https://github.com/eclipse/lemminx/milestone/44?closed=1) (May 27, 2024)
+
+### Enhancements
+
+ * XML Files support with settings. See [#1464](https://github.com/eclipse/lemminx/issues/1464).
```

Comment:
I think we can come up with a more descriptive title for this feature, maybe this:

Setting to specify which attributes should be treated as filepaths for the purpose of editor features